juniper/Cargo.toml

```diff
@@ -28,6 +28,7 @@ async = ["juniper_codegen/async", "futures"]
 expose-test-schema = []
 default = [
     "chrono",
+    "chrono-tz",
```

Comment:
I'm not sure this meets the level for on by default. Need to think about this some more.

Reply:
fair point. I have removed it from the default feature list